dependencies.props

```diff
@@ -47,17 +50,18 @@
     
     <XmlUpdateStep Include="CoreFx">
       <Path>$(MSBuildThisFileFullPath)</Path>
-      <ElementName>PlatformPackageVersion</ElementName>
-      <PackageId>Microsoft.NETCore.Platforms</PackageId>
+      <ElementName>CoreFxExpectedPrerelease</ElementName>
```

Comment:
Let's not use pre-release properties.  They make it hard to handle servicing and RTM packages.  See @dagood's last paragraph in https://github.com/dotnet/core-setup/pull/2243#issuecomment-298700379.